tools/ci_build/github/azure-pipelines/bigmodels-ci-pipeline.yml

```diff
@@ -214,7 +214,7 @@ stages:
             python3 -m pip install /Release/*.whl; \
             pushd /workspace/onnxruntime/python/tools/transformers/models/stable_diffusion; \
             python3 -m pip install -r requirements-cuda11.txt; \
-            python3 -m pip install --upgrade polygraphy onnx-graphsurgeon --extra-index-url https://pypi.ngc.nvidia.com; \
+            python3 -m pip install --upgrade polygraphy onnx-graphsurgeon ; \
```

Comment:
The onnx-graphsurgen from pypi is a placeholder:
   https://pypi.org/project/onnx-graphsurgeon
   
It's fine right now since the CUDA EP used in the test does not rely on the package. If we want to test TRT EP in the future, this will cause issue.